lib/src/kratos_client.dart

```diff
@@ -195,11 +196,11 @@ class KratosClient {
     if (effectiveIdToken == null) {
       SdkResult? sdkResult;
 
-      if (Platform.isAndroid &&
+      if (!kIsWeb &&
```

Comment:
This changes the meaning - previously, this `if` could be taken on Android, now it will be taken on all non-web platforms. The same goes for the next case.

As I understand it, the intention is to run Google/Apple SDK sign in only on the platforms that these are supported on, and fallback to web-based auth on the rest of them. This will change that.

Reply:
Yes you are correct, I've misunderstood the logic here. Returned platform checks back just after the `!kIsWeb` check to avoid unavailability errors.